lib/request.js

```diff
@@ -103,9 +103,9 @@ class AmazonMwsRequest {
 
     getResult(data) {
     	if (this.options.data)
-	        const path = `${this.options.basePath}.${this.options.data}`;
+	        var path = `${this.options.basePath}.${this.options.data}`;
```

Comment:
yep it's better (I suggest you to surround the conditional expression with parentesis)